docs/samples/Microsoft.ML.Samples/Dynamic/SsaSpikeDetectorTransform.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// the alignment of the usings with the methods is intentional so they can display on the same level in the docs site.
```

Comment:
Please, can you delete the copyright, since you are on the file?

There is another workitem to do so for all files:  #1576